### ISSUE TITLE
Customize mobile invitation service

### ISSUE THREAD

**Issue by chamerling** [jitsi/jitsi-meet#5503]
From what I see in the React code, we can define `inviteServiceUrl` which will be used to send invitations to a custom service. This is great!

Here are some questions:

1. According to what I understand from the code, this is also used in the mobile application, can someone confirm this? ie the web `config.js` file is used in the mobile app when opening a conference?
2. If 1 is true, how can (or is it possible) I update/configure to allow user to fill emails in the form instead of awaiting for phone number?

Thanks a lot

**Comment by saghul** [comment]
>     1. According to what I understand from the code, this is also used in the mobile application, can someone confirm this? ie the web `config.js` file is used in the mobile app when opening a conference?

Yep, mobiles also load config.js

>     2\. If 1 is true, how can (or is it possible) I update/configure to allow user to fill emails in the form instead of awaiting for phone number?

This is not implemented. I was thinking about this the other day, but there are somee complications: the dialog that shows up is designed to use a backend to do this. If it were emails only we could do it client side though. Something to think about.

**Comment by chamerling** [comment]
@saghul So, can we use something like a dumb REST service which sends back email compliant data from user input? 

ie from https://github.com/jitsi/jitsi-meet/blob/e683d70a18bbda97c25b6cdb6a5eb366a63ec270/react/features/invite/components/add-people-dialog/native/AddPeopleDialog.js, there is a `_query` call which is supposed to send back search results. If we are able to plug some REST API call here so that the REST service sends back items with right `type` and then call `inviteServiceUrl`, then we are almost good.

WDYT?

**Comment by saghul** [comment]
@chamerling Yep, that would be one way to go. But I think we'd need to make the supported entries configurable, since not all backends may implement all options.

**Comment by chamerling** [comment]
So is it possible to do this currently as a dirty hack while having this more clean? Are we able to configure a service being called by `_query`? I do not get where is `_query` comes from and how I can change it

**Comment by saghul** [comment]
Yes, you can implement it yourself I think. Alas we don'tt really have any documentation on it. Here is what the query function does: https://github.com/jitsi/jitsi-meet/blob/e683d70a18bbda97c25b6cdb6a5eb366a63ec270/react/features/invite/functions.js#L133 those URLs are what you needd to set in config.js and we'll hit those as the user types.

**Comment by chamerling** [comment]
Oh sorry, did not read the `_query` function until the end. Thanks for it, I will try to make it work as is and write this somewhere.

Thanks again for your help
